src/htdocs/index.php

```diff
@@ -1,10 +1,107 @@
 <?php
 
 if (!isset($TEMPLATE)) {
-  $TITLE = "GeoServe";
+  include_once '../conf/config.inc.php';
+
+  $TITLE = "Nearby Places";
   include_once 'template.inc.php';
 }
 
+$latitude = param('latitude', null);
+$longitude = param('longitude', null);
+$limit = param('limit', 5);
+if (!$limit) {
+  $limit = null;
+}
+$distance = param('distance', null);
+if (!$distance) {
```

Comment:
When you submit the form without a value, it is the empty string instead of null.